File: 프로그래머스/0/181830. 정사각형으로 만들기/정사각형으로 만들기.java
class Solution {
    public int[][] solution(int[][] arr) {
        int n = Math.max(arr.length, arr[0].length);
        int[][] ret = new int[n][n];
        for(int i = 0; i < n; i++){
            for(int j = 0; j < n; j++){
                if(i >= arr.length || j >= arr[0].length) continue;
                ret[i][j] = arr[i][j];
            }
        }
        return ret;
    }
}

File: 프로그래머스/0/181830. 정사각형으로 만들기/정사각형으로 만들기.py
def solution(arr):
    n = max(len(arr), len(arr[0]))
    print(n)
    ret = [[0] *n for _ in range(n)]
    for i in range(len(arr)):
        for j in range(len(arr[i])):
            ret[i][j] =  arr[i][j]
    return ret